tests/validate/bandwidth_test/src/host.cpp

```diff
@@ -54,9 +54,9 @@ int main(int argc, char** argv) {
         std::cout << "ERROR : please provide the platform test path to -p option\n";
```

Comment:
This main() function is huge with multiple duplicate sections,  please consider refactoring it into smaller functions.